docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -735,28 +563,86 @@ As database IDs are all globally unique, it's fine to replace table ID with data
 
 It's same for partitioned tables.
 
-### Future plans
 
-Many other features in TiDB are in development, some of which may influence placement rules.
+### Rule priorities
 
-Clustered index affects the key format of primary index. Fortunately, the prefix of key range is untouched.
+Tables only inherit rules from databases when they are created, and the value is saved in the meta data. Thus, the rules of priorities are simplified from an earlier version of this proposal (and are more inline with how character sets are inherited).
 
-Global secondary index largely affect the placement rules of partitioned tables. The key range of one global secondary index is not successive, so if it's necessary to define placement rules on the index, multiple rules should be generated in the PD. But for now, there's no such scenario.
+The only rules are that indexes and partitions inherit the rules of tables. Partitions can explicitly overwrite the placement policy, but indexes currently can not.
```

Comment:
Maybe clarify that indexes are not possible to define placement rules for, they will be places together with their table data (table or partition)?